_specs/ecip-1000.md

```diff
@@ -75,9 +75,12 @@ The current ECIP editors are:
 * Wei Tang (@sorpaas)
 * Isaac Ardis (@meowsbits)
 * Cody Burns (@realcodywburns)
-* Afri Schoedon (@soc1c)
 * Yaz Khoury (@YazzyYaz)
 
+The following ECIP editors have decided to remain pesedo-anonymous, and we will only be referring to them using Github handles:
```

Comment:
```suggestion
```

This seems unnecessarily verbose. Pseudonymity is implied when only a handle is present.

Reply:
Hmm I actually still think it's important to distinguish anonymous and non-anonymous editors, because to be honest, anonymous editors are not common, if you look into BIP and EIP process, none of them currently have anonymous editors.